### PR TITLE
add Stigg to bitbucket feature

### PR DESCRIPTION
**Pull request by barshimi**
Close: #5684

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Review by morhag90 — APPROVED** [review]

File: packages/amplication-client/src/util/BillingFeature.ts [hunk 1]
```diff
@@ -7,4 +7,5 @@ export enum BillingFeature {
   CodePushToGit = "feature-code-push-to-git",
   IgnoreValidationCodeGeneration = "feature-ignore-validation-code-generation",
   HideNotifications = "feature-ignore-validation-code-generation-hide-notifications",
+  BitbucketProvider = "feature-bitbucket",
```
Comment (abrl91) [inline packages/amplication-client/src/util/BillingFeature.ts:10]: We have a kind of convention here
```suggestion
  Bitbucket = "feature-bitbucket",
```

File: packages/amplication-server/src/core/billing/billing.types.ts [hunk 2]
```diff
@@ -15,4 +15,5 @@ export enum BillingFeature {
   ServicesWithManyEntities = "feature-services-wth-many-entities",
   SmartGitSync = "feature-smart-git-sync",
   TeamMembers = "feature-team-members",
+  BitbucketProvider = "feature-bitbucket",
```
Comment (abrl91) [inline packages/amplication-server/src/core/billing/billing.types.ts:18]:
```suggestion
  Bitbucket = "feature-bitbucket",
```

File: packages/amplication-server/src/core/git/git.provider.service.ts [hunk 3]
```diff
@@ -595,6 +598,17 @@ export class GitProviderService {
   ): Promise<GitOrganization> {
     const { code, gitProvider, workspaceId } = args.data;
 
+    const bitbucketEntitlement = this.billingService.isBillingEnabled
+      ? await this.billingService.getBooleanEntitlement(
+          workspaceId,
+          BillingFeature.BitbucketProvider
```
Comment (abrl91) [inline packages/amplication-server/src/core/git/git.provider.service.ts:604]:
```suggestion
          BillingFeature.Bitbucket
```

File: packages/amplication-client/src/Resource/git/GitActions/GitProviderConnectionList.tsx [hunk 4]
```diff
@@ -33,6 +35,11 @@ export const GitProviderConnectionList: React.FC<Props> = ({
   onProviderSelect,
 }) => {
   const { trackEvent } = useTracking();
+  const { stigg } = useStiggContext();
+
+  const showBitbucketConnect = stigg.getBooleanEntitlement({
+    featureId: BillingFeature.BitbucketProvider,
```
Comment (abrl91) [inline packages/amplication-client/src/Resource/git/GitActions/GitProviderConnectionList.tsx:41]:
```suggestion
    featureId: BillingFeature.Bitbucket,
```

Comment (abrl91) [inline packages/amplication-client/src/Resource/git/GitActions/GitProviderConnectionList.tsx:41]:
```suggestion
    featureId: BillingFeature.Bitbucket,
```

**Review by abrl91 — CHANGES_REQUESTED** [review]

**Review by abrl91 — APPROVED** [review]

**Pull request merged**